src/ecephys_etl/modules/align_timestamps/channel_states.py

```diff
@@ -16,12 +16,11 @@ def extract_barcodes_from_states(
         Sample index of each event.
     sampling_rate : numeric
         Samples / second
-    **barcode_kwargs : 
+    **barcode_kwargs :
```

Comment:
Can we at least refer the user to a different docstring that explicitly defines the allowed/expected kwargs?

Reply:
I agree, I think this needs to be explicit, or typed out somehow.